sound/soc/sof/intel/hda-stream.c

```diff
@@ -747,6 +732,12 @@ void hda_dsp_stream_free(struct snd_sof_dev *sdev)
 	if (bus->posbuf.area)
 		snd_dma_free_pages(&bus->posbuf);
 
+#if IS_ENABLED(CONFIG_SND_SOC_SOF_HDA)
+	/* free position buffer */
+	if (bus->rb.area)
+		snd_dma_free_pages(&bus->rb);
+#endif
+
```

Comment:
Can you split this RB part to a fix in a separate patch, this is not controversial and can be merged without issues.

Reply:
Sure, Rander is doing this(#280 ) so let me remove this part.